src/main.css

```diff
@@ -14,3 +14,29 @@
 body {
   background: url("./assets/stars.png"), #0a0a19;
 }
+ /* ===== Scrollbar CSS ===== */
+   * {
+    scrollbar-width: thin;
```

Comment:
Can you make both (firefox and chromium) scrollbars the same width? The Firefox one is too small. In fact it would be good if they are both changed to 5px.